### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mhauru**
With @dsj976, we updated the main README. An update to the figure in it is still forthcoming.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,25 +22,27 @@ The digital twin will:
...
-- **5** The Unity 3D model is found in [this repo](http://github.com/alan-turing-institute/CROP_unity): 
+- The CROP web application is the main interface for the digital twin. Users can
+    - explore collected heterogeneous IoT sensor data,
+    - analyse farm conditions at various points in time,
+    - use the interactive 3D visualisation of the farm,
+    - forecast future farm conditions using machine learning models built into the platform.
+- The CROP database is constantly updated from multiple streams of data: Hyper API, Stark energy usage platform, and others.
+- For forecasting, we CROP uses two models
+    - An ARIMA model uses past temperature and relative humidity data in the farm to forecast conditions a few days into the future.
+    - A GES model uses past sensor data, weather data, farm operational parameters (lighting schedules, fan settings, etc.) and Gaussian processes to forecast conditions a few days into the future. The GES model has the ability to predict various alternative scenarios, such as how would conditions change if the lights were switched on at a different time, or the fan settings were changed.
```
Comment (nbarlowATI) [inline README.md:33]: Maybe here I would restructure the sentence slightly, to separate the method from the inputs, e.g. "The GES model uses ... ,... to forecast conditions a few days into the future, using Gaussian Process Regression."  ?
(Also, do we know what "GES" stands for ?)

File: README.md [hunk 2]
```diff
@@ -89,20 +91,50 @@ CROP is implemented using a well established software stack (given below) and ex
...
+* Build a Docker container for the webapp, and push it to Docker Hub.
+* Build a Docker container for the Azure function apps that collect data into the database (ingress functions), and push it to Docker Hub.
+* Publish the Azure function app for running the forecasting models.
+* Run the CROP test suite.
 
-#### Manual
+The Azure services for the webapp and the ingress functions listen to updates on Docker Hub, and deploy the latest container once it has been pushed to the Hub.
 
-Steps of how to set up and deploy CROP locally or on Azure are documented [here](https://github.com/alan-turing-institute/CROP/wiki/1.-Set-up-and-deploy-CROP-locally-or-on-Azure).
+`main` and `develop` are deployed to production and testing versions of the platform, respectively. The former is relatively stable, the latter may be broken in the course of development.
```
Comment (nbarlowATI) [inline README.md:137]: Maybe "The `main` and `develop` branches ..." ?

**Review by nbarlowATI — APPROVED** [review]
Looks great - much better!!

At some point we should probably add a full Contributing.md page - the statement on this page about submitting a PR to this repo is perhaps misleading because only people with Write access to the repo can do that...

But I think the sooner we merge this (and also into `main`), the better!

File: README.md (hunk 2 above)
Comment (dsj976) [reply README.md:137]: Done

**Pull request merged**